tools/config-crd-ref-docs.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Google LLC
```

Comment:
This is the configuration file for our crd-ref-docs tool used to generate `docs/api.md`